protocol/Cargo.toml

```diff
@@ -18,7 +18,19 @@ thiserror = "1.0.56"
 [features]
 debug = ["std"]
 default = ["std", "debug", "symbols", "all_devices"]
-all_devices = ["device_2x4hd", "device_msharc4x8", "device_10x10hd", "device_4x10hd", "device_shd", "device_ddrc24", "device_nanodigi2x8", "device_ddrc88bm", "device_c8x12v2", "device_m2x4", "device_flex"]
+all_devices = [
+    "device_2x4hd",
+    "device_msharc4x8",
+    "device_10x10hd",
+    "device_4x10hd",
+    "device_shd",
+    "device_ddrc24",
+    "device_nanodigi2x8",
+    "device_ddrc88bm",
+    "device_c8x12v2",
+    "device_m2x4",
+    "device_flex",
+]
```

Comment:
Should the newly added `device_flex_ht` be listed here?

Reply:
Yes, added